<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Ressources/ressources.qrc">
    <normaloff>:/Icones/icone_Application.png</normaloff>:/Icones/icone_Application.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>938</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Fichier">
    <property name="title">
     <string>&amp;Fichier</string>
    </property>
    <addaction name="actionNouveau"/>
    <addaction name="actionOuvrir"/>
    <addaction name="actionEnregistrer"/>
    <addaction name="actionExporter_vers_une_image"/>
    <addaction name="separator"/>
    <addaction name="actionFermer_l_onglet"/>
    <addaction name="actionQuitter"/>
   </widget>
   <widget class="QMenu" name="menuEdition">
    <property name="title">
     <string>Edition</string>
    </property>
    <addaction name="actionRedimensionner_l_Algorithme"/>
    <addaction name="actionRenommer_l_algorithme"/>
   </widget>
   <widget class="QMenu" name="menuG_n_rer">
    <property name="title">
     <string>Générer</string>
    </property>
    <addaction name="actionVers_Ada"/>
    <addaction name="actionVers_Java"/>
    <addaction name="actionVers_C_C"/>
   </widget>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>Aide</string>
    </property>
    <addaction name="actionManuel"/>
    <addaction name="separator"/>
    <addaction name="actionA_propos_de_Tabula_Rasa"/>
    <addaction name="actionA_propos_de_Qt"/>
   </widget>
   <widget class="QMenu" name="menuAnalyse">
    <property name="title">
     <string>Analyse</string>
    </property>
    <addaction name="actionV_rifier_les_cycles"/>
   </widget>
   <addaction name="menu_Fichier"/>
   <addaction name="menuEdition"/>
   <addaction name="menuAnalyse"/>
   <addaction name="menuG_n_rer"/>
   <addaction name="menuA_propos"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMode_Insertion"/>
   <addaction name="actionMode_Edition"/>
   <addaction name="actionRoi"/>
   <addaction name="separator"/>
   <addaction name="actionAction"/>
   <addaction name="actionProcedure"/>
   <addaction name="actionSortie"/>
   <addaction name="separator"/>
   <addaction name="actionIteration"/>
   <addaction name="actionCondition"/>
   <addaction name="actionConditionMultiple"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNouveau"/>
   <addaction name="actionOuvrir"/>
   <addaction name="actionEnregistrer"/>
   <addaction name="actionExporter_vers_une_image"/>
  </widget>
  <action name="actionMode_Insertion">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/pointer.png</normaloff>:/Icones/pointer.png</iconset>
   </property>
   <property name="text">
    <string>Mode &amp;Insertion</string>
   </property>
   <property name="toolTip">
    <string>Permet de déplacer les éléments</string>
   </property>
  </action>
  <action name="actionMode_Edition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/linepointer.png</normaloff>:/Icones/linepointer.png</iconset>
   </property>
   <property name="text">
    <string>Mode &amp;Edition</string>
   </property>
   <property name="toolTip">
    <string>Permet de créer une hiérarchie entre les éléments</string>
   </property>
  </action>
  <action name="actionIteration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_boucle.png</normaloff>:/Icones/vignette_boucle.png</iconset>
   </property>
   <property name="text">
    <string>Itération</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une itération</string>
   </property>
  </action>
  <action name="actionCondition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_condition.png</normaloff>:/Icones/vignette_condition.png</iconset>
   </property>
   <property name="text">
    <string>condition</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une condition</string>
   </property>
  </action>
  <action name="actionAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_action.png</normaloff>:/Icones/vignette_action.png</iconset>
   </property>
   <property name="text">
    <string>Action</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une action</string>
   </property>
  </action>
  <action name="actionProcedure">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_procedure.png</normaloff>:/Icones/vignette_procedure.png</iconset>
   </property>
   <property name="text">
    <string>procedure</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer un appel de procedure</string>
   </property>
  </action>
  <action name="actionConditionMultiple">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_conditionMultiple.png</normaloff>:/Icones/vignette_conditionMultiple.png</iconset>
   </property>
   <property name="text">
    <string>ConditionMultiple</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une condition multiple</string>
   </property>
  </action>
  <action name="actionRoi">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_roi.png</normaloff>:/Icones/vignette_roi.png</iconset>
   </property>
   <property name="text">
    <string>Roi</string>
   </property>
   <property name="toolTip">
    <string>Un roi pour les gouverner tous</string>
   </property>
  </action>
  <action name="actionSortie">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/vignette_sortie.png</normaloff>:/Icones/vignette_sortie.png</iconset>
   </property>
   <property name="text">
    <string>Sortie</string>
   </property>
   <property name="toolTip">
    <string>Permet d'insérer une action de sortie</string>
   </property>
  </action>
  <action name="actionNouveau">
   <property name="text">
    <string>Nouveau</string>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ouvrir</string>
   </property>
  </action>
  <action name="actionEnregistrer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Enregistrer</string>
   </property>
  </action>
  <action name="actionExporter_vers_une_image">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/exporter_Image.png</normaloff>:/Icones/exporter_Image.png</iconset>
   </property>
   <property name="text">
    <string>Exporter vers une image</string>
   </property>
  </action>
  <action name="actionFermer_l_onglet">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/fermer_Onglet.png</normaloff>:/Icones/fermer_Onglet.png</iconset>
   </property>
   <property name="text">
    <string>Fermer l'onglet</string>
   </property>
  </action>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/fermer_Application.png</normaloff>:/Icones/fermer_Application.png</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
  <action name="actionRedimensionner_l_Algorithme">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_resize.png</normaloff>:/Icones/icone_resize.png</iconset>
   </property>
   <property name="text">
    <string>Redimensionner l'Algorithme</string>
   </property>
  </action>
  <action name="actionV_rifier_les_cycles">
   <property name="text">
    <string>Vérifier les cycles</string>
   </property>
  </action>
  <action name="actionVers_Ada">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/ada.jpg</normaloff>:/Icones/ada.jpg</iconset>
   </property>
   <property name="text">
    <string>Vers Ada</string>
   </property>
  </action>
  <action name="actionVers_Java">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/java.png</normaloff>:/Icones/java.png</iconset>
   </property>
   <property name="text">
    <string>Vers Java</string>
   </property>
  </action>
  <action name="actionVers_C_C">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/c.png</normaloff>:/Icones/c.png</iconset>
   </property>
   <property name="text">
    <string>Vers C/C++</string>
   </property>
  </action>
  <action name="actionA_propos_de_Tabula_Rasa">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_Application.png</normaloff>:/Icones/icone_Application.png</iconset>
   </property>
   <property name="text">
    <string>A propos de Tabula Rasa</string>
   </property>
  </action>
  <action name="actionA_propos_de_Qt">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/qt.png</normaloff>:/Icones/qt.png</iconset>
   </property>
   <property name="text">
    <string>A propos de Qt</string>
   </property>
  </action>
  <action name="actionManuel">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Manuel</string>
   </property>
  </action>
  <action name="actionRenommer_l_algorithme">
   <property name="icon">
    <iconset resource="../Ressources/ressources.qrc">
     <normaloff>:/Icones/icone_rename.png</normaloff>:/Icones/icone_rename.png</iconset>
   </property>
   <property name="text">
    <string>Renommer l'algorithme</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../Ressources/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
